### PR TITLE
feat: use default grpc parameters for grpc connection pool connection

### PR DESCRIPTION
**Pull request by samsja**
# Context

According to this https://github.com/grpc/grpc/blob/master/doc/keepalive.md we need to setup the client in the same way as the server for the keepalive parameters to work

fixing : https://github.com/jina-ai/jina/issues/5135

# What this pr do:

- [x] add the default grpc param to the connection pool

### CODE REVIEW

**Review by alaeddine-13 — COMMENTED** [review]
what about options that are specific to server only, they have no effect on client right ?

**Comment by samsja** [comment]
> what about options that are specific to server only, they have no effect on client right ?

Yes you are right I should separate the concern

File: jina_cli/autocomplete.py [hunk 1]
```diff
@@ -203,6 +204,7 @@
             '--version',
             '--loglevel',
```
Comment (JoanFM) [inline jina_cli/autocomplete.py:205]: why are these changes related?

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: jina_cli/autocomplete.py (hunk 1 above)
Comment (samsja) [reply jina_cli/autocomplete.py:205]: let me check this come from our best friend jina bot I believe

Comment (alaeddine-13) [reply jina_cli/autocomplete.py:205]: I think with upstream dependencies and more args added in jcloud, jina bot infers it and adds it to our autocomplete

Comment (alaeddine-13) [reply jina_cli/autocomplete.py:205]: here they added logout https://github.com/jina-ai/jcloud/pull/84

Comment (samsja) [reply jina_cli/autocomplete.py:205]: good catch

**Review by alaeddine-13 — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]
